### PR TITLE
Get public IP for Azure vmss nodes

### PR DESCRIPTION
**Pull request by feiskyer**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Get public IP for Azure vmss nodes.
```

/kind bug
/sig azure

### CODE REVIEW

**Comment by feiskyer** [comment]
cc @kkmsft

File: pkg/cloudprovider/providers/azure/azure_vmss.go [hunk 1]
```diff
@@ -295,8 +295,30 @@ func (ss *scaleSet) GetIPByNodeName(nodeName string) (string, string, error) {
 		return "", "", err
 	}
 
-	targetIP := *ipConfig.PrivateIPAddress
-	return targetIP, "", nil
```
[the remark this answers is not in the document]
Comment (feiskyer) [reply pkg/cloudprovider/providers/azure/azure_vmss.go]: Good catch. The comment should be removed.

**Review by andyzhangx — APPROVED** [review]
/lgtm

**Pull request merged**